### ISSUE TITLE
Add Chainstack as RPC Provider

### ISSUE THREAD

**Issue by MinnytheRapper** [oasisprotocol/docs#426]
Chainstack is now integrated with Oasis and they would like us to add them to our official docs as an RPC provider

I was thinking something similar to how Polygon does it : https://wiki.polygon.technology/docs/operate/network-rpc-endpoints/ ?

Relevant links:
https://console.chainstack.com/user/account/create
https://chainstack.com/build-better-with-oasis-sapphire/
https://chainstack.com/protocols/oasis-sapphire/
https://docs.chainstack.com/docs/oasis-sapphire-tooling
https://docs.chainstack.com/docs/oasis-sapphire-tutorial-understanding-confidential-smart-contracts-with-oasis-sapphire

**Comment by matevz** [comment]
> I was thinking something similar to how Polygon does it : https://wiki.polygon.technology/docs/operate/network-rpc-endpoints/ ?

It says 404 not found.

Ok, found it: https://wiki.polygon.technology/docs/pos/reference/rpc-endpoints/

Under the Infrastruture providers they don't say directly what's the URL a dApp user can connect to though. Any many of them are dead links too.

**Comment by wilwixq** [comment]
That is ok. Again this is for DApps mostly, not end users. Just acknowledging that there is a high performance 3rd party option and linking them to the https://chainstack.com/protocols/oasis-sapphire/ the same way polygon does is fine. Also they link to chainlist which provides all available public RPC endpoints, but again this is for DApps so that does not really matter, but we can probably link to Chainlist as well because why not and then as we potentially add more public RPCs to chainlist it won't require any updates on our end. 

So basically my thoughts would be add something like 

Infrastructure Providers
Public RPCs may have rate limits or traffic restrictions. For a dedicated 3rd party RPC without such restrictions, please see the following providers

Chainstack (https://chainstack.com/build-better-with-oasis-sapphire/) 

For a complete list of public endpoints, visit
Chainlist (https://chainlist.org/?search=Oasis+Sapphire)

**Comment by CedarMist** [comment]
1rpc also supports Oasis Sapphire

**Comment by matevz** [comment]
This was added in #668.
